docs/source/conf.py

```diff
@@ -98,7 +98,6 @@
 # html_theme_options = {}
 
 # Add any paths that contain custom themes here, relative to this directory.
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 # html_theme_path = []
```

Comment:
I'd prefer to keep those commented lines in place since they are a form of documentation. They might be helpful in the future if there's a need to customize the docs.